go.mod

```diff
@@ -4,8 +4,8 @@ go 1.13
 
 require (
 	github.com/anz-bank/pkg v0.0.7
-	github.com/arr-ai/frozen v0.13.0
-	github.com/arr-ai/hash v0.4.0
+	github.com/arr-ai/frozen v0.14.0
+	github.com/arr-ai/hash v0.5.0
```

Comment:
Since you're bumping versions anyway, may as well grab wbnf v0.19.0.

Reply:
whoops thought I did